packages/contractkit/src/test-utils/ganache.setup.ts

```diff
@@ -3,7 +3,18 @@ import * as ganache from '@celo/ganache-cli'
 import * as path from 'path'
 
 const MNEMONIC = 'concert load couple harbor equip island argue ramp clarify fence smart topic'
-
+export const ACCOUNT_PRIVATE_KEYS = [
```

Comment:
what are these for?

Reply:
They are the corresponding private keys for the accounts in ganache. They are the same from the protocol tests. Sometimes we need access to the private keys to do some of these actions, like comparing if signatures are equivalent. Though this would also meanz that we can now get ridd of the private keys in the protocol tests since we can just "natively" sign